tensornetwork/backends/abstract_backend.py

```diff
@@ -734,3 +734,28 @@ def matmul(self, tensor1: Tensor, tensor2: Tensor) -> Tensor:
     """
     raise NotImplementedError(
         "Backend '{}' has not implemented `matmul`.".format(self.name))
+  
+  def pivot(self, tensor: Tensor, pivot_axis: int = 1) -> Tensor:
```

Comment:
Is this a standard name / do we really need this over just `reshape`?

Reply:
It's a fair question. The main use case is as part of other functions that properly act on matrices. Here we see it used in SVD, QR, and RQ (although I haven't actually refactored accordingly), and as we add more decompositions we'll need to repeat it more times. So it seemed to me a potential way to compress code.

Reply:
sgtm.